lib/src/client.rs

```diff
@@ -16,6 +16,8 @@ use nimiq_genesis::NetworkInfo;
 use nimiq_light_blockchain::LightBlockchain;
 #[cfg(feature = "validator")]
 use nimiq_mempool::mempool::Mempool;
+#[cfg(feature = "validator")]
```

Comment:
Maybe this shouldn't be part of this feature

Reply:
We don't have a better feature flag right now, this should be addresses as part of #2719